docs/Expo-and-Ignite.md

```diff
@@ -0,0 +1,55 @@
+# Expo and Ignite
+
+![ignite+expo](https://miro.medium.com/max/1400/1*Ii4JuTWmVLeVBcqFyX3v5g.jpeg)
+
+> Expo is the easiest, nicest experience for Ignite apps. And I am convinced Ignite (especially the upcoming version 8, Maverick) is the nicest boilerplate for Expo apps you can possibly get. - Jamon Holmgren
+
+If you're not familiar with [Expo](https://expo.dev), it's an open-source platform for making universal native apps for Android, iOS, and the web with [React Native](https://reactnative.dev). It's essentially a series of layers built around React Native. Here's a partial list of what Expo is:
+
+- [Expo Go](https://expo.dev/client) -- an app for quickly previewing React Native apps without building them
+- [EAS](https://eas.dev) -- a build & distribution service for mobile apps, especially RN
+- A _lot_ of high quality, well-maintained [React Native libraries](https://github.com/expo/expo)
+- Several integrated services, such as push notifications, over-the-air updates, [and much more](https://docs.expo.dev/)
+
+## Overview
+
+In previous versions of Ignite (versions 6 and 7), you could pass in a `--expo` flag to make the resulting generated app "Expo-ready". In version 8 (code-named "Maverick") we made the boilerplate "Expo-ready" by default -- but without locking you into using Expo Go or Expo's services if you don't want to. Every new Ignited app is ready to run with Expo CLI or with "vanilla" React Native CLI.
+
+To run your new Ignite app in Expo CLI (and launch Expo Go in a simulator/emulator):
+
+```
+yarn expo:start
+yarn expo:ios
+yarn expo:android
+```
```

Comment:
TOL: it may be nice to also include the command start to finish

```
npx ignite-cli new PizzaApp
cd PizzaApp
yarn expo:start
```

Then have a screenshot or something for starting the emulator. However, that may be out of scope.

Reply:
Good call!